### ISSUE TITLE
7.0 hr_holidays_extension: missing employee_id arg in is_public_holiday()

### ISSUE THREAD

**Issue by alexis-via** [OCA/hr#66]
In hr_holidays_extension, in the method onchange_bynumber() on line 198, there is:

```
public_holiday = holiday_obj.is_public_holiday(cr, uid, next_dt.date(), context=context)
```

But, in hr_public_holidays, in is_public_holiday() line 63, the proto is:

```
def is_public_holiday(self, cr, uid, dt, employee_id=None, context=None)
```

and, if employee_id = None, then it will only search for hr.holidays.public where country_id = False. This is a bug :-(

**Comment by alexis-via** [comment]
@miketelahun it seems this bug is in your code. What do you think it this issue and the patch I propose ?

**Comment by yvaucher** [comment]
fixed by #67
